HOWTO_SNAP.md

```diff
@@ -0,0 +1,61 @@
+Steps:
+
+- First check snapcraft is currently building something fine:
+
+```
+snapcraft
+```
+
+- Install it and try it, something like the following, replacing 
+  for what you just built:
```

Comment:
replacing the snap name with the one you just built

Reply:
thanks